frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -1,38 +1,60 @@
 import React from "react";
 import { Card, Button, Row, Col } from "react-bootstrap";
-import cowHead from "./../../../assets/CowHead.png"; 
+import cowHead from "./../../../assets/CowHead.png";
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
-    // update cowPrice from fixture
+const ManageCows = ({ userCommons, commons, onBuy, onSell }) => {
     return (
         <Card>
-        <Card.Header as="h5">Manage Cows</Card.Header>
-        <Card.Body>
-            {/* change $10 to info from fixture */}
```

Comment:
I believe this shouldn't be here anymore

Reply:
I believe it isnt, this is one of the removed lines